src/function/scalar/string/hex.cpp

```diff
@@ -165,6 +165,210 @@ static void ToHexFunction(DataChunk &args, ExpressionState &state, Vector &resul
 	}
 }
 
+static idx_t NumberOfLeadingingZeros(uint64_t x) {
```

Comment:
There is a builtin method for this, we use it in `chimp_utils.hpp`, you could probably use the `CountZeros::Leading` method defined there

Reply:
I also want to use built-in functions, but I don't found the use of built-in functions in the core code base. :thumbsup: